### PR TITLE
Fixed doctrine:generate:crud issue when not using id as primary key (issue #545)

### PR DESCRIPTION
**Pull request by mohdshakir**

### CODE REVIEW

**Comment by mohdshakir** [comment]
PR created;
https://github.com/sensiolabs/SensioGeneratorBundle/pull/546

**Comment by fabpot** [comment]
@hhamon @HeahDude Can one of you validate these changes?

File: Resources/skeleton/crud/actions/delete.php.twig [hunk 1]
```diff
@@ -6,7 +6,7 @@
      *
 {% block phpdoc_method_annotations %}
 {% if 'annotation' == format %}
-     * @Route("/{id}", name="{{ route_name_prefix }}_delete")
+     * @Route("/{{ '{' }}{{ identifier }}{{ '}' }}", name="{{ route_name_prefix }}_delete")
```
Comment (HeahDude) [inline Resources/skeleton/crud/actions/delete.php.twig]:
```
{{ '{' ~ identifier ~ '}' }}
```
would be more readable than:
```
{{ '{' }}{{ identifier }}{{ '}' }}
```
and same many times below.

File: Resources/skeleton/crud/actions/delete.php.twig [hunk 2]
```diff
@@ -42,7 +42,7 @@
     private function createDeleteForm({{ entity_class }} ${{ entity_singularized }})
     {
         return $this->createFormBuilder()
-            ->setAction($this->generateUrl('{{ route_name_prefix }}_delete', array('id' => ${{ entity_singularized }}->getId())))
+            ->setAction($this->generateUrl('{{ route_name_prefix }}_delete', array('{{ identifier }}' => ${{ entity_singularized }}->get{{ identifier | capitalize }}())))
```
Comment (HeahDude) [inline Resources/skeleton/crud/actions/delete.php.twig]: Can you remove spaces around filter pipe?

**Comment by mohdshakir** [comment]
Thanks @HeahDude for the comments. I've updated accordingly.

**Comment by fabpot** [comment]
Thank you @mohdshakir.

**Pull request closed without merging**